resources/assets/js/app/views/topics/topicDetail.js

```diff
@@ -1,5 +1,37 @@
 var Marionette = require('backbone.marionette');
+var Bookmark = require('../../models/BookmarkModel');
 
 module.exports = Marionette.ItemView.extend({
-    template: 'topicDetail'
+    template: 'topicDetail',
+    events: {
+        'click #bookmark-btn': 'addBookmark'
```

Comment:
please create a property 'ui' and add button id there. And example:
ui: {
   bookmarkButton: '#bookmark-btn'
}

then you can work with it like: this.ui.bookmarkButton. Or '@ui.bookmarkButton' in events.
Change all of the using